exercism/diamond/diamond.js

```diff
@@ -0,0 +1,15 @@
+var Diamond = function() {}
+
+Diamond.prototype.makeDiamond = function(letter){
+  var alphabet = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z']
+
+var space = ' '
+var leadingSpace = function() {
+  alphabet.indexOf(letter) - index of that letter
```

Comment:
Yes, but the second part is pseudocode. Here's a reference we checked after we saw your question. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf